Week1/homework/app.js

```diff
@@ -1,11 +1,187 @@
-'use strict';
+/* eslint-disable no-unused-expressions */
+{
+  ('use strict');
+  let bookTitles = [
+    'deep_work',
+    'anna_frank',
+    'forty_rules',
+    'memory_in_flesh',
+    'dust_of_promises',
+    'art_of_forgetting',
+    'black_suits_you',
+    'Chaos_Of_Senses',
+    'bed_hopper',
+    'game_of_thrones',
+  ];
+  console.log(bookTitles);
```

Comment:
Remember to remove `console.log` if you don't need them in the pull request.

Reply:
Thanks for your tip! I just want to practice the q on another way, and keep it to remember it later  when I need it, I thought if I did not call the function then it will not work, Is that right?

Reply:
Not sure if I understand your question, the `console.log` will always be outputted since it is not inside a function.